cmd/vic-machine/configure/configure.go

```diff
@@ -217,6 +238,39 @@ func updateSessionEnv(sess *executor.SessionConfig, envName, envValue string) {
 	sess.Cmd.Env = newEnvs
 }
 
+func (c *Configure) processCertificates(conf *config.VirtualContainerHostConfigSpec) error {
+
+	if !c.certificates.NoTLSverify && (c.certificates.Skey == "" || c.certificates.Scert == "") {
+		log.Infof("No certificate regeneration requested. No new certificates provided. Certificates left unchanged.")
+		return nil
+	}
+
+	if c.certificates.CertPath == "" {
+		c.certificates.CertPath = c.DisplayName
+	}
+
+	_, err := os.Lstat(c.certificates.CertPath)
+	if err == nil || os.IsExist(err) {
+		return fmt.Errorf("Specified or default certificate output location \"%s\" already exists. Specify a location that does not yet exist with --cert-path to continue or do not specify --tls-noverify if, instead, you want to load certificates from %s", c.certificates.CertPath, c.certificates.CertPath)
+	}
+
+	var debug int
```

Comment:
Do we store this `int*` as an `int` somewhere? Otherwise, it needs to be passed in so that `ProcessCertificates` can print the Leaf of the cert if debug level >2. That functionality could also be stripped, in order to eliminate the need to pass in `debug` at all.